cmd/spire-server/cli/entry/delete.go

```diff
@@ -55,21 +55,26 @@ func (d DeleteCLI) Run(args []string) int {
 		return d.printErr(err)
 	}
 
-	cl, err := util.NewRegistrationClient(config.RegistrationUDSPath)
+	srvCl, err := util.NewServerClient(config.RegistrationUDSPath)
```

Comment:
nit: in order to normalize all our CLI, maybe you can implement [Adapter](https://github.com/spiffe/spire/blob/master/cmd/spire-server/util/util.go#L97)  that will provides you with a server client and defer it for youy at the same time it provides help, "default" configurations, and allow you to return errors and it will take care of printing it